docs/en/sql-reference/functions/untuple-functions.md

```diff
@@ -0,0 +1,106 @@
+---
+toc_priority: 66
+toc_title: Untuple
+---
+
+# Untuple Functions {#untuple-functions}
+
+## Untuple {#untuple}
+
+Performs syntactic substitution of [tuple](../../sql-reference/data-types/tuple.md#tuplet1-t2) elements in the call location.
+
+**Syntax**
+
+``` sql
+untuple(x)
+```
+
+You can use the `EXCEPT` expression to skip columns as a result of the query.
+
+**Parameters**
+
+-   `x` - A `tuple` function, column, or tuple of elements. [Tuple](../../sql-reference/data-types/tuple.md).
+
+**Returned value**
+
+-   None.
+
+**Examples**
+
+Input table:
+
+``` text
+┌─key─┬─v1─┬─v2─┬─v3─┬─v4─┬─v5─┬─v6────────┐
+│   1 │ 10 │ 20 │ 40 │ 30 │ 15 │ (33,'ab') │
+│   2 │ 25 │ 65 │ 70 │ 40 │  6 │ (44,'cd') │
+│   3 │ 57 │ 30 │ 20 │ 10 │  5 │ (55,'ef') │
+│   4 │ 55 │ 12 │  7 │ 80 │ 90 │ (66,'gh') │
+│   5 │ 30 │ 50 │ 70 │ 25 │ 55 │ (77,'kl') │
+└─────┴────┴────┴────┴────┴────┴───────────┘
+```
+
+Example of using a `tuple` function as a parameter `untuple` function:
```

Comment:
This is the very strange example that looks like a test case. I think we can delete it. It doesn't give a user valuable information.